ansible/roles/dashmate/tasks/main.yml

```diff
@@ -173,8 +173,20 @@
   changed_when: true
   when: dashmate_branch is defined
 
-# TODO: we should use status command to figure out if dashmate is running
-#   to get the status from the response
+- name: Get status
+  ansible.builtin.command: "{{ dashmate_cmd }} status --format=json"
+  become: true
+  become_user: dashmate
+  args:
+    chdir: '{{ dashmate_cwd }}'
+  register: dashmate_status
+  tags:
+    - skip_ansible_lint
```

Comment:
Why do we skip lint on this task?

Reply:
Its complaing about no-changed-when and that command should not change anything, but its not changing anything, it just registers the variable